src/pages/grants/grantsPage/components/GrantApplicantCard.tsx

```diff
@@ -0,0 +1,263 @@
+      return (
+        <Button
+          onClick={onOpen}
+          height="40px"
+          width="100%"
+          size="md"
+          textTransform="uppercase"
+          fontFamily={fonts.livvic}
+          variant="primary"
+        >
+          {t('Vote')}
+        </Button>
+      )
+    }
+
+    if (grantStatus !== GrantStatusEnum.Closed) {
+      return (
+        <Button as={Link} to={getPath('project', project.name)} size={'sm'} variant={'primary'}>
+          {t('View project')}
+        </Button>
+      )
+    }
+  }
+
+  return (
+    <CardLayout p={2} key={project.id}>
+      <Box display="flex">
+        <Box mr={3} height={{ base: '90px', lg: '144px' }}>
+          <Link
+            to={projectLink}
+            className={classNames(classes.image, isMobile ? classes.mobileImage : classes.desktopImage)}
+          >
+            <ImageWithReload
+              objectFit="cover"
+              borderRadius="7px"
+              width={isMobile ? '90px' : '144px'}
+              height={isMobile ? '90px' : '144px'}
+              src={project.thumbnailImage || ''}
+              alt="project thumbnail"
+            />
+          </Link>
+        </Box>
+        <Box pr={2} flexGrow={1}>
+          <Link to={projectLink}>
+            <H3 fontSize="18px">{project.title}</H3>
+          </Link>
+          <Link to={projectLink}>
+            <Text noOfLines={4} wordBreak="break-word">
+              {project.shortDescription}
+            </Text>
+          </Link>
+        </Box>
+        {!isMobile && (
+          <Box
+            minWidth="166px"
+            pr={4}
+            display="flex"
+            flexDirection="column"
+            justifyContent="center"
+            alignItems="center"
+            gap={2}
+          >
+            {(grantHasVoting || isClosed) && renderWidgetItem(funding, contributorsCount)}
+            {renderButton(project)}
+          </Box>
+        )}
+      </Box>
+      {contributors && contributors.length > 0 && (
+        <Box pl={2} filter="opacity(0.4)">
+          {contributors.map(
+            (contributor) =>
+              contributor && (
+                <AvatarElement
+                  key={contributor.user?.id}
+                  width="28px"
+                  height="28px"
+                  wrapperProps={{
+                    display: 'inline-block',
+                    marginLeft: '-5px',
+                    marginTop: 2,
+                  }}
+                  avatarOnly
+                  borderRadius="50%"
+                  seed={contributor?.user?.id}
+                  user={contributor?.user}
+                />
+              ),
+          )}
+        </Box>
+      )}
+      {isMobile && (
+        <VStack w="full">
+          {(grantHasVoting || isClosed) && renderWidgetItem(funding, contributorsCount)}
+          {renderButton(project)}
+        </VStack>
+      )}
+      <Modal isOpen={isOpen} onClose={onClose} title={t('How voting works')}>
```

Comment:
The Grant has a field called `votingSystem`, which returns an enum. Could you make this section dynamic based on the value of that enum? As other voting systems will have other descriptions.

Reply:
Totally, I will ask Mick for the copy we want displayed for the proportional voting system!